samples/api-client/manager/manager_test.py

```diff
@@ -53,8 +53,12 @@ def clean_up_registries():
         registry_id = registry.id
         if registry_id.find('test-registry-') == 0:
             time_str = registry_id[
-                registry_id.rfind('-') + 1: len(registry_id)]
-            test_date = datetime.datetime.utcfromtimestamp(int(time_str))
+                registry_id.rfind('-') + 1 : len(registry_id)]
+            try:
+                test_date = datetime.datetime.utcfromtimestamp(int(time_str))
+            except ValueError:
+                # skip incorrect formatted registry
+                continue
             now_date = datetime.datetime.utcfromtimestamp(int(time.time()))
```

Comment:
It looks like `test_date` will not be set if there is an error.

Reply:
Does `continue` skip to the next registry and ignore rest of code inside of loop ? 
I wanted to skip registry with wrong formatted name.

Reply:
Ah I didn't notice this was in a loop. My bad! 🤦‍♀️